lca-cli/postpivot/postpivot.go

```diff
@@ -38,7 +38,6 @@ import (
 
 	cp "github.com/otiai10/copy"
 	"github.com/sirupsen/logrus"
-	etcdClient "go.etcd.io/etcd/client/v3"
```

Comment:
If we move https://github.com/openshift-kni/lifecycle-agent/blob/1365cbe57129b75b964006fc17b4f34ed5a84f65/lca-cli/ops/ops.go#L123-L142 to recert as well we can get rid of the etcd lca dependency altogether, if we care about that

Reply:
Actually why not , though here was the idea to run it only on seed in order not to block installation. I believe we can add flag to recert and run it on demand in this case

Reply:
but this will be next pr